### ISSUE TITLE
Basic tools support (pre-defined brushes support)

### ISSUE THREAD

**Issue by olekolek1000** [liquidev/rkgk#96]
Allow the user to select pre-defined tools like brush, floodfill, rectangle, lines (...). It would be a good idea to use haku language itself to do this.
Blocked by https://github.com/liquidev/rkgk/issues/32

**Comment by liquidev** [comment]
I think this could be solved with a library of built-in brushes that do exactly that, once we have #32 resolved.

One thing that came up in my head is that it would be interesting to add a (non-destructive?) way to transform any dotter brush into a different reticle (#12), so that e.g. you can trace out a perfect rectangle with any brush.

But that's just an idea for now, it's not needed for us to provide a basic brush library :smile:
